Packs/IntegrationsAndIncidentsHealthCheck/pack_metadata.json

```diff
@@ -44,6 +44,10 @@
         "MailSenderNew": {
             "mandatory": false,
             "display_name": "Mail Sender (New)"
+        },
+        "Demisto REST API": {
+            "mandatory": false,
```

Comment:
@altmannyarden @abaumgarten should we make it `true`?

Reply:
yes

Reply:
great, I'll change